lib/rules/method.js

```diff
@@ -35,19 +54,7 @@ module.exports = {
                 case "Identifier":
                 case "MemberExpression":
                     if ("property" in node.callee && node.arguments.length > 0) {
```

Comment:
Yeah most of the pr comment is out of date since updates I ran. However I wanted to ask if we can drop this side of the filter. I think it's original code but will check when back in the office.

The net effect will be we can work on bare functions called `write` for example which I wanted to highlight. Should object filtering/matching restrict bare functions or should it be a different config prop?